config/purposes/scrna_core_library_prep.wip.yml

```diff
@@ -0,0 +1,106 @@
+# This configuration file is for the plate and tube purposes required for the
+# high throughput scRNA Core library prep pipeline
+---
+# There will be a manual submission on a sub-selection of samples in the LRC GEM-X 5p cDNA PCR XP and
+# LRC GEM-X 5p cDNA Input plates for aggregation into a single LRC GEM-X 5p Cherrypick plate. Multiple plates
+# could be aggregated into a single cherrypick plate.
+# e.g. if single samples are selected from each plate then up to 10 aggregate plates could be made which are
+# then combined into a single cherrypick plate.
+LRC GEM-X 5p Aggregate:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate: false
+  :creator_class: LabwareCreators::TenStamp
+  :file_links: []
+# NB. QC concentrations need to be uploaded via QuantHub for the Cherrypick plate in order for the binning
+# calculations in the Dil plate step to work.
+# TODO: there will be a submission on this plate for library prep
+# TODO: not clear what the stock and input flags should be for this to work
+LRC GEM-X 5p Cherrypick:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate: false
+  :cherrypickable_target: true
+  :creator_class: LabwareCreators::TenStamp
+  :state_changer_class: StateChangers::AutomaticPlateStateChanger
+  :work_completion_request_type: 'limber_scrna_core_chromium_gem_x_5p_aggregation'
+  :file_links:
+    - name: 'Download Hamilton Aggregate Cherrypick CSV'
+      id: 'hamilton_aggregate_cherrypick'
+    - name: Download Concentration (ng/ul) CSV
+      id: concentrations_ngul
+# NB. For the following binning step the min and max amounts for the bins are the total amount of cDNA
+# in the well i.e. QC concentration * (source_volume + diluent_volume)
+# Here we are using a fixed source volume and there is no dilution (diluent_volume = 0)
+# TODO: R&D to confirm the number of PCR cycles for each range
+LRC GEM-X 5p Dil:
```

Comment:
Took 'GEX' out of this name as it wasn't present for any of the others.

Reply:
I think in the bespoke pipelines this was because the Dil plate onwards is where we've decided on GEX, TCR or BCR routes and those purposes and pipelines had the choice in the purpose names for the Lab staff to know which path the plates were on. Check with Abby?

e.g in future here we will likely need TCR and BCR variants of all these. 

Also might be worth considering the lengths of the purpose names as they'll be 4 characters longer for those (incl space), so including GEX now means no length change for testing labels.

Reply:
Agreed, if we have TCR and BCR variants we will have to create purposes for those.  In the Bespoke 5p pipeline, 'GEX' is in all of them.

I think we took it out of these ones because 
a) it was confusing next to 'GEM-X', and 
b) makes for longer label text (although like you say, the TCR/BCR variants will be that length when/if they come along). 

But the 'gene expression' part is still implied in all of these. I didn't think it made sense being in the Dil plate but not the others.
We could add it into all of them? If so I'd suggest using 'GE' as Abby suggested for the library type name, to make it a bit more distinct than 'GEM-X'.